csharp/src/Microsoft.AI.MachineLearning.Interop/Microsoft.AI.MachineLearning.props

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this file needed for nuget?

Reply:
This is needed so that consumers of the nuget are able to reference the Microsoft.AI.MachineLearning.winmd in their project.

The Samples Gallery uses this to generate a native component that works with the NuGet WinML Types.